src/makielayout/blocks/colorbar.jl

```diff
@@ -68,7 +68,12 @@ function Colorbar(fig_or_scene, contourf::Union{Contourf, Tricontourf}; kwargs..
 
 end
 
-colorbar_range(start, stop, length, _) = LinRange(start, stop, length)  # noop
+colorbar_range(start, stop, length, _::typeof(identity)) = LinRange(start, stop, length)
+function colorbar_range(start, stop, length, _)
+    @warn "cannot determine inverse_transform"
```

Comment:
Not sure what this does, but I'm almost never fond of warnings. I'd rather have a clear error if we hit edge cases

Reply:
This is when an automatic scaling has been applied, but we cannot determine the inverse transform for the colorbar ticks, hence falling back to identity, which leads to un-optimal ticks. I got tricked by this, and I think a warning is thus useful. Maybe it should advise to use `Makie.ReversibleScale` instead ...

Reply:
Yeah, I think this is a good place for a warning, _but_ only with a good advice on how to resolve the warning ;)

Reply:
So, is the `maybe use Makie.ReverseibleScale instead ?` hint added in https://github.com/MakieOrg/Makie.jl/pull/3095/files#diff-b49aa74de270f83f7d96f661d23e3e1044de710184415a64e5db70890a671841R95 sufficient ?

Reply:
Maybe:

`@warn "Cannot determine inverse transform: you can use colorscale=Makie.ReversibleScale($(_), reverse($(_))) instead"`
I love warnings that give context + and basically just need a copy and paste to be fixed.
Also, shouldn't be this somewhere else in the stack?
So that we get a similar warning for ` inverse_transform(scale)` ? Maybe to complicated for now, so just musing ^^
I guess one could do:
```julia
function colorbar_range(start, stop, length, scale)
    rev = inverse_transform(scale)
    if isnothing(rev)
          @warn "Cannot determine inverse transform: you can use `colorscale=Makie.ReversibleScale($(_), reverse($(_)))` instead. Falling back to identity, which leads to sub optimal ticks!"`
          rev = identity
    end
    rev.(range(start, stop; length))
end
```
This would also keep out the union type from `colorbar_range` which _may be_ desirable for compilation (likely not a big issue).

Reply:
Oh yeah, inverting the logic here also allows us to get rid on the `REVERSIBLE_SCALES` global `Union`, better !

I've implemented the changes in the latest commit.